java/arcs/android/host/AndroidHost.kt

```diff
@@ -14,20 +14,19 @@ import android.content.Context
 import androidx.lifecycle.Lifecycle
 import arcs.core.host.ArcHost
 import arcs.core.host.ParticleRegistration
-import arcs.jvm.host.AnnotationBasedJvmProdHost
-import arcs.jvm.host.JvmProdHost
+import arcs.core.host.Resurrector
+import arcs.jvm.host.JvmHost
 import arcs.sdk.android.storage.ServiceStoreFactory
-import java.util.ServiceLoader
 
 /**
- * An [ArcHost] that runs isolatable particles that are expected to have no platform
- * dependencies directly on Android APIs. Automatically scans class path using
- * [ServiceLoader] to find additional particles.
+ * An [ArcHost] runs on Android inside of a [Service].
  */
-class AndroidProdHost(
+open class AndroidHost(
```

Comment:
It's only open so that test hosts can extend it, but non-tests can use it directly.  This class was meant to be used 'out of the box' if desired, but I can make it abstract.

Reply:
Done (abstract)